src/react/index.js

```diff
@@ -13,4 +13,5 @@ export { default as ErrorBoundary } from './ErrorBoundary';
 export { default as ErrorPage } from './ErrorPage';
 export { default as LoginRedirect } from './LoginRedirect';
 export { default as PageWrap } from './PageWrap';
-export { useAppEvent } from './hooks';
+export { useAppEvent, useParagonTheme } from './hooks';
+export { paragonThemeActions } from './reducers';
```

Comment:
[sanity check / clarification] Can you expand on why we've want to export  `paragonThemeActions` to consumers? I believe the intent for consumers to manage switching theme variants is to use the `paragonTheme.setThemeVariant` function exposed within `AppContext`?

When will consumers use the exported `paragonThemeActions`?

Reply:
@brian-smith-tcril to look into if this has been updated and resolve

Reply:
looks like this was addressed in https://github.com/openedx/frontend-platform/commit/e3211d9a7200dc7fe63cec5992ccea3522f30fdb